source/_docs/nested-docroot.md

```diff
@@ -16,12 +16,12 @@ This is especially useful for third party dependencies, such as those installed
 
 <div class="alert alert-info" role="alert">
 <h4 class="info">Note</h4>
-<p>One-click Dashboard updates are not compatible with the nested <code>web</code> docroot, and you'll need to manually maintain core updates. We recommend using a <a href="https://pantheon.io/docs/custom-upstream/">Custom Upstream</a>.</p>
+<p>One-click Dashboard updates are only compatible with the nested <code>web</code> docroot feature when the `web_docroot` property is set in the `pantheon.upstream.yml` file of the upstream. We recommend using a <a href="https://pantheon.io/docs/custom-upstream/">Custom Upstream</a> if you would like to continue to utilize Dashboard updates.</p>
 </div>
 
 ## One-Time Setup
 
-Below we recommend using Git, but you can also use SFTP to set your site up for nested docroot.
+If you wish to stop using One-click Dashboard updates, and instead intend to update your site with Composer, then you may create a site with a nested docroot by adjusting your `pantheon.yml` file. Below we recommend using Git, but you can also use SFTP to set up your site.
```

Comment:
This feels wrong. Should we not be recommending they switch upstreams rather than committing themselves to a never-ending stream of useless updates in the dashboard?